### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by shobanamg**

### CODE REVIEW

File: javascript/javascript1/week2/classwork/bank.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  console.log("Please deposit some money!");
+} else if (balance > 0 && balance <= 1000) {
+  console.log("Your balance is looking okay");
+} else if (balance > 1000 && balance <= 3000) {
+  console.log("Your balance is looking good");
+} else if (balance > 3000 && balance <= 10000) {
+  console.log("Your balance is fantastic");
+} else {
+  console.log("Your balance is Amazing");
+}
```
Comment (mutaquihadev) [inline javascript/javascript1/week2/classwork/bank.js:13]: Maybe you could simplify the comparison, by just checking if(balance < VALUE).
If statements only jump to the next if the previous comparison evaluate as false

File: javascript/javascript1/week2/classwork/count-down.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      console.log(i);
+    } else if (i === 0) {
+      console.log("Happy new year!");
+    } else {
+      console.log("it is not a new year");
+    }
+  }
+}
+
+countDown(10);
```
Comment (mutaquihadev) [inline javascript/javascript1/week2/classwork/count-down.js:15]: I like your implementation, maybe we can discuss some improvements!

File: javascript/javascript1/week2/classwork/countby-sound.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "Terri",
+  "Phil",
+  "Lola",
+  "Sam",
+  "Kay",
+  "Bruce",
+];
+
+for (i = 0; i < names.length; i++) {
+  if (names[i].toLowerCase().includes("a") === true) continue;
```
Comment (mutaquihadev) [inline javascript/javascript1/week2/classwork/countby-sound.js]: Includes already returns a boolean, I think the comparison can be accomplished without the check with ===true

**Review by mutaquihadev — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week2/classwork/count-down.js (hunk 2 above)
Comment (shobanamg) [reply javascript/javascript1/week2/classwork/count-down.js:15]: Sure Mutaq.. l would like to know what can be improved. Let me know when you have time.

**Pull request merged**